test-loop-tests/src/tests/fix_min_stake_ratio.rs

```diff
@@ -1,162 +0,0 @@
-use crate::setup::builder::TestLoopBuilder;
-use crate::setup::env::TestLoopEnv;
-use crate::utils::ONE_NEAR;
-use crate::utils::validators::get_epoch_all_validators;
-use itertools::Itertools;
-use near_async::messaging::SendAsync;
-use near_async::test_loop::data::TestLoopData;
-use near_async::time::Duration;
-use near_chain_configs::test_genesis::{TestGenesisBuilder, ValidatorsSpec};
-use near_network::client::ProcessTxRequest;
-use near_o11y::testonly::init_test_logger;
-use near_primitives::epoch_manager::EpochConfigStore;
-use near_primitives::hash::CryptoHash;
-use near_primitives::num_rational::Rational32;
-use near_primitives::test_utils::create_user_test_signer;
-use near_primitives::transaction::SignedTransaction;
-use near_primitives::types::AccountId;
-use near_primitives::upgrade_schedule::ProtocolUpgradeVotingSchedule;
-use near_primitives::version::ProtocolFeature;
-use std::string::ToString;
-use std::sync::atomic::{AtomicU64, Ordering};
-
-/// Check that small validator is included in the validator set after
-/// enabling protocol feature `FixMinStakeRatio`.
-#[test]
-fn slow_test_fix_min_stake_ratio() {
```

Comment:
(aside) Wouldn't it be better to keep this test, and simplify it? Removing the part about voting for new protocol version, just checking that the small validator is part of the validator group

Reply:
(aside) I tried doing that and after spending an hour plus, I couldn't get it working...